lib/ansible/modules/service.py

```diff
@@ -419,7 +419,7 @@ def service_enable_rcconf(self):
 
             # Write out the contents of the list into our temporary file.
             for rcline in new_rc_conf:
-                os.write(TMP_RCCONF, rcline)
+                os.write(TMP_RCCONF, rcline.encode())
```

Comment:
Prefer our `to_bytes` helper here.

```suggestion
                os.write(TMP_RCCONF, to_bytes(rcline, errors='surrogate_or_strict'))
```

Reply:
@relod Thank you for your review. I did commit your suggestion (84c822cfe2e54addb822d8b052e47442b44597cb) , and added changelog fragment (2bf14a9f2ec8830a273c2e92ad84bd3931bdd41d).